Docs/Help/Restore-Bicep.md

```diff
@@ -0,0 +1,60 @@
+---
+external help file: Bicep-help.xml
+Module Name: Bicep
+online version:
+schema: 2.0.0
+---
+
+# Restore-Bicep
+
+## SYNOPSIS
+Restores external modules from the specified Bicep file to the local module cache.
+
+## SYNTAX
+
+```
+Restore-Bicep [-Path] <String> [<CommonParameters>]
+```
+
+## DESCRIPTION
+Restores external modules from the specified Bicep file to the local module cache.
```

Comment:
can we improve details around caching? sources we retrieve from, where is the local cache, etc.